### PR TITLE
Support us added below contact form

### LINKED ISSUE #200: Support Us Section
Below the Contact Form, add a section for patreon, buymeacoffee and relevant text+illustrations about Support US

### PR DESCRIPTION
**Pull request by paraschhugani**
made Support us below contact form and added a quote for open source too , 
 
Closes #200 

can i add myself in contribution team in main.js, i think the bonus you gave me in pull request #262 was for this !

Screenshot 2020-10-10 at 12 45 32 AM

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Sure. Feel free to add your name to the contributor section

File: index.html [hunk 1]
```diff
@@ -155,6 +155,41 @@ <h3><span id="confirm"></h3>
...
+        <div id="Support" style="margin-top: 60px;margin-left:10%;">
+            <h3 style="color:rgb(0, 153, 255)">The power of Open Source is the power of the people. The people rule. <small style="color: gray;">-Philippe Kahn</small></h3>
+            <div style="margin-top: 10px;padding-left: 15vw; font-size: 1.5vw;">
+            <h1>Support Us and help us help you.</h1>
+            <ul>
+                <div class="patreonAndCoffe" style="margin-top: 10px;margin-left: 10%;">
+                    <a id="patreonLink" href="https://www.patreon.com/smaranjitghose" target="_blanck"></a>
+                    <div id="patreon" style="" onclick="document.getElementById('patreonLink').click();">
+                        <img style="width: 40px;" src="https://github.githubassets.com/images/modules/site/icons/funding_platforms/patreon.svg" alt="">
+                        <span style="color:blue;"><u>patreon.com/smaranjitghose</u></span>
```
Comment (smaranjitghose) [inline index.html:168]: Avoid using inline CSS as much as possible

**Comment by smaranjitghose** [comment]
mobile_respons
# Feedback:
- Can you please make this more mobile responsive. For example: Reduce the size of the quote and the Support us text so that we can have each of them in a single line
- Add some space between the quote and the "Support..."
- The icons themselves are sufficient. Put anchor tags to redirect to the respective websites for funding
- Place both the icons in a single line
- Try experimenting with a micro animation or a hover effect for the icons

**Comment by paraschhugani** [comment]
i have take the suggestions in mind and made the changes accordingly , making new pr

**Pull request closed without merging**